onboarding/values-staging.yaml

```diff
@@ -0,0 +1,64 @@
+---
+
+global:
+  registry: ''
+
+onboarding:
+  replicaCount: 1
+
+  images:
+    repository: docker.io/italia/developers-italia-onboarding
+    tag: latest
+    pullPolicy: Always
+
+  deployEnvironment: "staging"
+
+  env:
+    onboarding_port: "3000"
+    onboarding_environment: "staging"
+    onboarding_app_base_url: "https://onboarding-staging.zland.org"
+    onboarding_es_url: "https://elasticsearch.developers.italia.it/indicepa_pec/_search"
+    onboarding_smtp_port: "587"
+    onboarding_smtp_secure: "false"
+    onboarding_email_from: "Team per la Trasformazione Digitale <contatti@developers.italia.it>"
+    onboarding_email_cc: ""
+    onboarding_email_bcc: "contatti@developers.italia.it"
+    onboarding_email_override_recipient_addr: "fabio.bonelli@teamdigitale.governo.it"
```

Comment:
We need a shared email address here

Reply:
I would use `contatti@developers.italia.it`